lesson02.md

```diff
@@ -0,0 +1,440 @@
+<!-- .slide: id="lesson2" -->
+
+# Basic Frontend - Spring 2022
+
+Lesson 2, Thursday, 2022-03-17
+
+---
+
+### Recap
+
+What did we talk about last lesson?
+
+---
+
+### Recap: data types
+
+Number:
+```js
+42
+-3.1415
+```
+
+String:
+```js
+"John"
+'Berlin'
+```
+
+Boolean:
+```js
+true
+false
+```
+
+---
+
+### Recap operators
+
+| Group | Operators | Example |
+| ----- | --------- | ------- |
+| Numerical Operators | `+ - * / **` | `5 + 4 * 3`<br/>`7 / 2 - 2`<br/>`"Hello" + " World"` |
+| Comparison Operators | `=== !== < > <= >=` | `30 !== 25`<br/>`20 >= 18` |
```

Comment:
All these operators in one line, next to each other are a bit confusing in the Slides. Especially for people that they look at the for the first time. Maybe we can put them in some kind of boxes or separate them in some way?